功能: 播放本地指定音乐并验证播放状态

  场景: 播放本地指定音乐并验证播放状态
    当 < 打开音乐应用 ... passed in 0.498s
    当 < 打开我的音乐库 ... passed in 0.736s
    当 < 播放本地指定音乐 ... passed in 14.056s
      | music_name |
      | 小城故事       |
    那么 < 验证音乐名称一致 ... passed in 0.689s
      | chk_music_name |
      | 小城故事           |
    那么 < 验证歌手名字一致 ... passed in 0.201s
      | chk_artist |
      | 邓丽君        |
    那么 < 验证音乐是否播放 ... passed in 4.783s
      | is_playing |
      | true       |

功能: 暂停音乐并验证播放状态暂停

  场景: 暂停音乐并验证播放状态暂停
    当 < 打开音乐应用 ... passed in 0.523s
    当 < 打开我的音乐库 ... passed in 1.042s
    当 < 播放本地指定音乐 ... passed in 2.601s
      | music_name |
      | 小城故事       |
    当 < 播放或暂停音乐 ... passed in 0.880s
    那么 < 验证音乐是否播放 ... passed in 4.231s
      | is_playing |
      | false      |

功能: 收音机暂停验证收音机状态

  场景: 收音机暂停验证收音机状态
    当 < 打开收音机应用 ... passed in 0.526s
    当 < 打开FM_AM选择界面 ... passed in 0.683s
    当 < 随机播放FM节目 ... passed in 5.553s
      | o_result |
      | o_name   |
    当 < 播放或暂停FM电台 ... failed in 13.913s
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/usr/local/lib/python3.4/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "steps/step_radio.py", line 258, in step_impl
    Radio().click_radio_pause_or_play_ele()
  File "/home/ogq/pateo/AT/ATP/autotestproject/actions/radio.py", line 185, in click_radio_pause_or_play_ele
    Utils().raise_Exception_info('上一首或下一首控件不存在')
  File "/home/ogq/pateo/AT/ATP/autotestproject/utils/utils.py", line 106, in raise_Exception_info
    raise Exception('用例运行失败，' + err_msg + '请参考截图信息: file:///' + png_path)
Exception: 用例运行失败，上一首或下一首控件不存在请参考截图信息: file:////home/ogq/pateo/AT/autotestLog/20160901/screenshots/20160901113853.png

Captured stdout:
/home/ogq/pateo/AT/autotestLog/20160901/screenshots/20160901113853.png
开始检查应用是否crash


功能: u盘音乐到蓝牙音乐

  场景: u盘音乐到蓝牙音乐
    当 < 打开音乐应用 ... passed in 0.506s
    当 < 打开我的音乐库 ... passed in 0.838s
    当 < 播放U盘音乐 ... passed in 1.854s
    那么 < 验证当前为U盘音乐 ... failed in 5.752s
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/site-packages/behave/model.py", line 1456, in run
    match.run(runner.context)
  File "/usr/local/lib/python3.4/site-packages/behave/model.py", line 1903, in run
    self.func(context, *args, **kwargs)
  File "steps/step_audio.py", line 216, in step_impl
    Utils().raise_Exception_info('当前音乐不是U盘音乐')
  File "/home/ogq/pateo/AT/ATP/autotestproject/utils/utils.py", line 106, in raise_Exception_info
    raise Exception('用例运行失败，' + err_msg + '请参考截图信息: file:///' + png_path)
Exception: 用例运行失败，当前音乐不是U盘音乐请参考截图信息: file:////home/ogq/pateo/AT/autotestLog/20160901/screenshots/20160901113916.png

Captured stdout:
/home/ogq/pateo/AT/autotestLog/20160901/screenshots/20160901113916.png
开始检查应用是否crash


